testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -767,6 +770,39 @@ public void testRenameTableAcrossSchema()
         assertFalse(getQueryRunner().tableExists(getSession(), renamedTable));
     }
 
+    @Override
+    public void testAddColumn()
+    {
+        if (!hasBehavior(SUPPORTS_ADD_COLUMN)) {
+            assertQueryFails("ALTER TABLE nation ADD COLUMN test_add_column bigint", "This connector does not support adding columns");
```

Comment:
Please make sure to run tests against untested connectors (BigQuery, Redshift, Glue) to make sure the `hasBehaviour` declarations are correct to prevent modifying the tpch tables state.

Reply:
it seems that `BaseConnectorTest` (nor `AbstractTestDistributedQueries`) is not implemented for BigQuery, Redshift or Glue

Reply:
Interesting. 😄